openbb_platform/extensions/equity/integration/test_equity_api.py

```diff
@@ -1131,8 +1131,8 @@ def test_equity_compare_groups(params, headers):
                 "adjustment": "splits_only",
                 "provider": "yfinance",
                 "symbol": "AAPL",
-                "start_date": "2023-06-01",
```

Comment:
I believe that this will make flaky tests if you set it to None as it won't match the cassette.

Reply:
There are no cassettes.